libc/cmake/modules/LLVMLibCObjectRules.cmake

```diff
@@ -41,6 +41,11 @@ function(_get_common_compile_options output_var flags)
     list(APPEND compile_options "-fno-unwind-tables")
     list(APPEND compile_options "-fno-asynchronous-unwind-tables")
     list(APPEND compile_options "-fno-rtti")
+    # clang-8+, gcc-12+
+    check_cxx_compiler_flag("-ftrivial-auto-var-init=pattern" LIBC_CC_SUPPORTS_PATTERN_INIT)
```

Comment:
Can you move this check outside of this function, as it is called quite a lot?  Either to the top of the file, or to somewhere in `CheckCompilerFeatures.cmake`.

Reply:
done in 862dc3c034909e68e971b15af95dfd106233f74e